packages/stargate/src/accounts.ts

```diff
@@ -40,49 +40,103 @@ function accountFromBaseAccount(input: BaseAccount): Account {
  */
 export type AccountParser = (any: Any) => Account;
 
+export type AccountParserRegistry = Map<Any["typeUrl"], AccountParser>;
```

Comment:
Realizing we may also want to export `accountFromBaseAccount`, as this will likely be needed by those supplying a custom account parser.  Alternatively, this could be baked into every `parseAccount` call as every AccountParser function likely needs this as the last step. 

Will wait for feedback on this and the overall design before making any updates.